WebDevStudios/wd_s
# Bump WP Scripts version

Closes #856

### DESCRIPTION
Updates to `wp-scripts` 23.0.0 and resolves some new linting issues.

The initial PR got a little out of whack and is throwing assertion errors, so I figured I'd create a clean branch off of `main` with the latest updates.

### OTHER


### STEPS TO VERIFY
1. Checkout
2. `npm i`
3. Run `start`, `watch`, `build`, `lint`, and `format` to test
4. Verify the frontend still looks good and works as expected

### DOCUMENTATION
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

May need to update documentation with updates around linting/formatting.
